tests/utils.go

```diff
@@ -1859,6 +1859,37 @@ func NewRandomVMIWithNS(namespace string) *v1.VirtualMachineInstance {
 	return vmi
 }
 
+func GetSupportedCPUFeature(node k8sv1.Node) string {
```

Comment:
@ksimon1 See my comment in the conversation

Reply:
I wanted to avoid code duplicate, because the same functions are called from two different files.

Reply:
@ksimon1 Oh I understand. Maybe for a matter of consistency return array of strings that represent non blacklisted models/features and in the test code take whatever item you want.